docs/content/guides/developer/getting-started/sui-install.mdx

```diff
@@ -29,28 +29,91 @@ brew install mystenlabs/tap/sui
 
 Each Sui release provides a set of binaries for several operating systems. You can download these binaries from GitHub and use them to install Sui.
 
+<Tabs groupId="operating-systems">
+
+<TabItem value="linux" label="Linux">
 1. Go to https://github.com/MystenLabs/sui.
 1. In the right pane, find the **Releases** section.
 
     ![Sui releases in GitHub](./images/releases.png)
 1. Click the release tagged **Latest** to open the release's page.
 1. In the **Assets** section of the release, select the .tgz compressed file that corresponds to your operating system.
 1. Double-click the downloaded file. If the file doesn't automatically expand, manually unzip the file.
-1. Open the expanded folder and double-click the appropriate binary to install, beginning with sui-`<OS>`-`<ARCHITECTURE>`:
+1. Open the expanded folder. Navigate to the `target/release` directory and double-click the appropriate binary to install, beginning with `sui-ubuntu-x86_64`:
```

Comment:
This won't install the binary, it'll just run it. For the `sui` binary, this would just open a terminal, show the help output, and exit. This is true on all platforms.

Reply:
Will test.